forge-gui/res/cardsfolder/upcoming/scholar_of_new_horizons.txt

```diff
@@ -3,11 +3,12 @@ ManaCost:1 W
 Types:Creature Human Scout
 PT:1/1
 K:etbCounter:P1P1:1
-A:AB$ ChangeZone | Cost$ T RemoveAnyCounter<1/Any/Permanent.YouCtrl/a permanent you control> | Origin$ Library | Destination$ Library | ChangeType$ Land.Plains | ChangeNum$ 1 | RememberChanged$ True | Reveal$ True | Shuffle$ False | SubAbility$ ToBattlefieldTapped | SpellDescription$ Search your library for a Plains card and reveal it. If an opponent controls more lands than you, you may put that card onto the battlefield tapped. If you don't put the card onto the battlefield, put it into your hand. Then shuffle.
-SVar:ToBattlefieldTapped:DB$ ChangeZone | ConditionCheckSVar$ Y | ConditionSVarCompare$ GTX | Defined$ Remembered | Optional$ True | Origin$ Library | Destination$ Battlefield | Tapped$ True | Shuffle$ True | SubAbility$ ToHand
-SVar:ToHand:DB$ ChangeZone | Defined$ Remembered | ConditionNotPresent$ Card.IsRemembered | Origin$ Library | Destination$ Hand | Shuffle$ True | SubAbility$ DBCleanup
+A:AB$ ChangeZone | Cost$ T RemoveAnyCounter<1/Any/Permanent.YouCtrl/a permanent you control> | Origin$ Library | Destination$ Library | ChangeType$ Plains | ChangeNum$ 1 | RememberChanged$ True | Reveal$ True | Shuffle$ False | SubAbility$ ToBattlefieldTapped | StackDescription$ SpellDescription | SpellDescription$ Search your library for a Plains card and reveal it. If an opponent controls more lands than you, you may put that card onto the battlefield tapped. If you don't put the card onto the battlefield, put it into your hand. Then shuffle.
+SVar:ToBattlefieldTapped:DB$ ChangeZone | ConditionCheckSVar$ Y | ConditionSVarCompare$ GTX | Defined$ Remembered | Optional$ True | Origin$ Library | Destination$ Battlefield | Tapped$ True | Shuffle$ True | SubAbility$ ToHand | StackDescription$ None | ForgetChanged$ True 
+SVar:ToHand:DB$ ChangeZone | Defined$ Remembered | ConditionNotPresent$ Card.IsRemembered | Origin$ Library | Destination$ Hand | Shuffle$ True | SubAbility$ DBCleanup | StackDescription$ None
```

Comment:
I think you should remove the ConditionNotPresent here now that you have ForgetChanged

Also remove the Shuffle from line 7... otherwise you could end up shuffling twice